js/turtledefs.js

```diff
@@ -90,7 +90,7 @@ function createDefaultStack() {
     console.log('language setting is ' + language);
     if (language === 'ja') {
         DATAOBJS =
-            [[0, 'start', screen.width / 3, 100, [null, 1, null]],
+            [[0, 'start', (screen.width / 3)-140, 100, [null, 1, null]],
```

Comment:
Not sure what this calculation is supposed to do, but it seems it is probably going to cause problems when the screen is narrow. I thought the plan was to move the blocks just to the right of the mouse. That would be something like:

(screen.width / 2) + 28

Reply:
The position of the block is static and is not changing with respect to the screen size.
It should be reactive so that it changes according to the screen size.
But I am not sure as to how to make it reactive here.Could you help me on this.

Reply:
I found that the position of the turtle is changing when the screen changes so even this block can be placed similar to that.